mvc/migration/migratingfrommvc5.rst

```diff
@@ -1,257 +1,244 @@
 
-Now open Startup.cs and modify it as follows:
+- Open the *Startup.cs* file and change the code to match the following:
 
-.. code-block:: c#
+.. literalinclude:: migratingfrommvc5/samples/WebApp1/src/WebApp1/Startup.cs
+	:language: c#
+	:emphasize-lines: 6, 14-22
+	:linenos:  
+	:lines: 11-32
+	:dedent: 4
 
-	public void ConfigureServices(IServiceCollection services)
-	{
-		services.AddMvc();
-	}
 
-	public void Configure(IApplicationBuilder app)
-	{
-		app.UseMvc(routes =>
-		{
-			routes.MapRoute(
-				name: "default",
-				template: "{controller=Home}/{action=Index}/{id?}");
-		});
-	}
+``UseStaticFiles`` adds the static file handlers. As mentioned previously, the ASP.NET runtime is modular, and you must explicitly opt in to serve static files. For more information, see `Application Startup <http://docs.asp.net/en/latest/fundamentals/startup.html>`_ and `Routing <https://docs.asp.net/en/latest/fundamentals/routing.html>`__.
 
-At this point we are ready to create a simple Controller and View. Add a Controllers folder and a Views folder to the project. Add an MVC Controller called HomeController.cs class to the Controllers folder and a Home folder in the Views folder. Finally, add an Index.cshtml MVC View Page to the Views/Home folder. The project structure should be as shown:
+Add a controller and view
+========================================
+
+In this section you'll add a minimal controller and view to serve as place holders for the MVC 5 controller and views you'll migrate in the next section.
+
+- Add a *Controllers* folder.
+- Add an MVC controller class with the name *HomeController.cs* to the *Controllers* folder.
+- Add a *Views* folder. 
+- Add and a *Views/Home* folder.
+- Add an *Index.cshtml* MVC view page to the *Views/Home* folder. 
+
+The project structure is shown below:
 
 .. image:: migratingfrommvc5/_static/project-structure-controller-view.png
 
-Modify Index.cshtml to show a welcome message:
+Replace the contents of the *Views/Home/Index.cshtml* file with the following:
 
 .. code-block:: html
 
 	<h1>Hello world!</h1>
 
-Run the application - you should see Hello World output in your browser.
+Run the app.
 
 .. image:: migratingfrommvc5/_static/hello-world.png
 
 .. _migrate-controllers-views-static:
 
-Migrate Basic Controllers, Views, and Static Content
-----------------------------------------------------
+See :doc:`../controllers/index` and :doc:`../views/index` for more information.
+
+Now that we have a minimal working MVC 6 project, we can start migrating functionality from the MVC 5 project. We will need to move the following:
+
+- client-side content (CSS, fonts, and scripts)
+- controllers
+- views
+- models
+- bundling 
+- filters
+- Log in/out, identity (This will be done in the next tutorial.)
```

Comment:
Shouldn't you capitalize the first character for each line in this list. Do did that on the last list.